### DISCUSSION TITLE
HTTP/2 How can I don't create a new Connection every Times?

### DISCUSSION THREAD

**Discussion by julyzd** [naver/ngrinder#766] [Q&A] [answered]
HTTP/2 How can I don't create a new Connection every Times?

**Comment by donggyu04** [comment, marked as answer]
@julyzd 

Hi, ngrinder run your test script during configured duration or running count.
After every running the script, ngrinder disconnect all connections that used in the script.
There is no option to disable it.

If you want to keep-alive request, using loop in your script.

> I will consider supporting keep-alive option in the future release.

**Reply by donggyu04** [reply]
@julyzd 

Relevant [feature](https://github.com/naver/ngrinder/pull/775) has been released.
